dockerfiles/Dockerfile-toolchain

```diff
@@ -64,15 +64,21 @@ RUN export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o /tmp/rust_install.sh
 RUN chmod +x /tmp/rust_install.sh
 RUN /tmp/rust_install.sh -y
-RUN export PATH="$PATH:$HOME/.cargo/bin"
-RUN echo 'export PATH="$PATH:$HOME/.cargo/bin"' >> /home/opam/.bashrc
+ENV PATH "$PATH:$HOME/.cargo/bin"
```

Comment:
Do not use $HOME, it's not always set properly. Use the exact path "/home/opam/"